macos-trust-cert.app/Contents/Info.plist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>CFBundleDisplayName</key>
+  <string>macos-trust-cert</string>
+  <key>CFBundleExecutable</key>
+  <string>add_certificate_test</string>
+  <key>CFBundleIdentifier</key>
+  <string>com.mitmproxy.mitmproxy</string>
```

Comment:
FWIW let's use `org.mitmproxy.cert_install` here, we don't really use mitmproxy.com.